civictechprojects/views.py

```diff
@@ -79,6 +79,8 @@ def group_tags_counts(request):
 
 # TODO: Pass csrf token in ajax call so we can check for it
 @csrf_exempt
+@api_view()
```

Comment:
Do we need the `@api_view()` lines?  If so, then POST methods like group_create need to be called with 'POST', because otherwise it defaults to just allowing GET.
```suggestion
@api_view(['POST'])
```

Reply:
From what I could find online, the @throttle_classes() decorator is meant to be used in conjunction with the @api_view() decorator and may not function correctly without it. I reviewed each of the views and updated the methods in the api_view() decorators, hopefully appropriately!